### PR TITLE
dev-ruby/websocket: initial commit

### PR DESCRIPTION
**Pull request by junghans**
In the dependency tree for https://github.com/travis-ci/travis.rb, @gentoo/ruby please review as ruby ebuilds are usually not my cup of tea.

### CODE REVIEW

File: dev-ruby/websocket/websocket-1.1.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+USE_RUBY="ruby20 ruby21"
+RUBY_FAKEGEM_RECIPE_DOC="none"
```
Comment (graaff) [inline dev-ruby/websocket/websocket-1.1.1.ebuild]: You could use RUBY_FAKEGEM_RECIPE_DOC="rdoc" here to generate documentation using a generic approach that does not depend on the gem.

File: dev-ruby/websocket/websocket-1.1.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+USE_RUBY="ruby20 ruby21"
```
Comment (graaff) [inline dev-ruby/websocket/websocket-1.1.1.ebuild]: I assume you don't have ruby22 installed and thus not tested with it. Normally we try to cover as many ruby targets as possible. We can also handle this when it lands in the tree.

**Comment by graaff** [comment]
Please make sure to test with FEATURES=test since the eclasses generate methods for that by default:

 * Running test phase for ruby20 ...
fatal: Not a git repository (or any parent up to mount point /var/tmp/portage)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
fatal: Not a git repository (or any parent up to mount point /var/tmp/portage)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
fatal: Not a git repository (or any parent up to mount point /var/tmp/portage)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
rake aborted!
Don't know how to build task 'test'

You'll want to use the eclass rspec support using RUBY_FAKEGEM_RECIPE_TEST="rspec3". This will also cover the rspec3 dependency, but not rspec-its since that is an add-on.

**Comment by monsieurp** [comment]
@gentoo/ruby

File: dev-ruby/websocket/websocket-1.1.1.ebuild (hunk 1 above)
Comment (junghans) [inline dev-ruby/websocket/websocket-1.1.1.ebuild]: Done!

**Comment by junghans** [comment]
Fixed `FEATURES=test`.

**Comment by junghans** [comment]
@gentoo/ruby : any further comments?

**Comment by monsieurp** [comment]
It looks good to go. @graaff?

**Comment by graaff** [comment]
:+1:

**Comment by junghans** [comment]
Thanks, 3 more to come!

**Pull request merged**